web/releases/nightly.json

```
@@ -115,83 +115,83 @@
       "filename": "index-katello.html",
       "guides": [
         {
-          "title": "Release Notes",
+          "title": "Release notes",
           "path": "Release_Notes"
         },
         {
-          "title": "Planning Guide",
+          "title": "Planning guide",
```

```suggestion
          "title": "Planning for Foreman with Katello",
```

See https://github.com/theforeman/foreman-documentation/pull/3195/files#diff-7c56a5e93aac7a02f5ff3b7c231a4573ce1dbd6a9d5f6e3c6979cf393d566431R54

---

Yes, what should I notice in particular?

---

Oh, I see now :)